src/components/RuleItem.vue

```diff
@@ -430,6 +431,55 @@ async function updateRuleShipping(event: any) {
     showToast(translate("Failed to update rule brokering."))
   }
 }
+
+function isReorderPossible(ruleDir: string, ruleIndex: any) {
+  if(ruleDir === 'prev' && ruleIndex === 0) return false;
```

Comment:
I think there is no need to check `prev` and `next` as if the rule is having an index of `0`, it simply means its the first rule and we can reorder it and similar case for the last rule.